modules/GenerateReport.nf

```
@@ -15,19 +18,26 @@ process GenerateReport {
     publishDir path: "${params.output_dir}", mode: 'copy'
     
     input:
-        file status_files
+        path status_files
     
     output:
-        file "consolidated_report.csv"
+        path "consolidated_report.csv"
     
     script:
     """
+    #!/bin/bash
+    set -euo pipefail
```

Can you explain why this command is needed?

---

`set -euo pipefail`
This is a set of shell options that make the script more robust and safer:
`-e`: Exit immediately if any command returns a non-zero status (error)
`-u`: Treat unset variables as errors and exit immediately
`-o pipefail`: Return the exit status of the last command in a pipeline that failed, rather than the last command overall

Added these in the workflow because:
1. We want to fail fast if any command fails (like if Almanack analysis fails)
2. We want to catch any undefined variables that might cause issues
3. We want to ensure pipeline failures (like | commands) are properly detected

Trying to avoid:
1. Continue running even after critical errors
2. Silently use empty values for undefined variables
3. Miss failures in pipeline commands

---

Awesome! Thanks